.github/workflows/requirements.yml

```diff
@@ -24,6 +24,7 @@ env:
   MLFLOW_CONDA_HOME: /usr/share/miniconda
   SPARK_LOCAL_IP: localhost
   PYTHON_VERSION: "3.10" # minimum supported version + 1
+  PIP_CONSTRAINT: ${{ github.workspace }}/requirements/constraints.txt
```

Comment:
LGTM! Can we add `- uses: ./.github/actions/show-versions` in both `skinny` and `core` jobs? The show-versions action allows maintainer to check what versions are installed in the current python environment.